app/views/home/index.html.slim

```diff
@@ -11,20 +11,17 @@
   .govuk-grid-column-one-half
     = m('home.about', headings_start_with: 'xl')
 
-  - unless current_user
-    .govuk-grid-column-one-half
-      .light-grey-box.enrol-box
```

Comment:
Are `.enrol-box` and `.white-space-pre-wrap` being used elsewhere or can they also be deleted?

Reply:
No, deleted now